hat/urls.py

```diff
@@ -17,15 +17,20 @@
 admin.site.site_title = "Iaso"
 admin.site.index_title = "Administration de Iaso"
 
+
 if settings.DISABLE_PASSWORD_LOGINS:
     urlpatterns = [
-        path("login/", TemplateView.as_view(template_name="iaso/disabled_password_login.html"), name="login"),
-        path("admin/login/", TemplateView.as_view(template_name="iaso/disabled_password_login.html"), name="login"),
+        path(
+            "admin/login/", TemplateView.as_view(template_name="iaso/disabled_password_login.html"), name="login-iaso"
+        ),
+        path("login/", TemplateView.as_view(template_name="iaso/disabled_password_login.html"), name="login-iaso"),
```

Comment:
Why did you change the order? Is it because reverse use the last occurence of the name "login-iaso"?